draft-ietf-quic-manageability.md

```diff
@@ -911,9 +911,24 @@ This document has no actions for IANA.
 
 # Security Considerations
 
-Supporting manageability of QUIC traffic inherently involves tradeoffs with the
-confidentiality of QUIC's control information; this entire document is therefore
-security-relevant.
+QUIC is an encrypted and authenticated transport. That means, once the
+cryptographic handshake is complete, QUIC endpoints discard most packets that
+are not authenticated, greatly limiting the ability of an attacker to interfere
+with existing connections.
+
+However, some information are still observerable, as supporting manageability of
+QUIC traffic inherently involves tradeoffs with the confidentiality of QUIC's
+control information; this entire document is therefore security-relevant.
+
+More security considerations for QUIC are discussed in {{!QUIC}} and {{!QUIC-TLS}},
+generally considering active or passive attackers in the network as well as attacks
+on specific QUIC mechanism.
+
+Version Negotiation packets do not contain any mechanism to prevent version downgrade
+attacks. However, future versions of QUIC that use Version Negotiation packets are
+require to define a mechanism that is robust against version downgrade attacks.
+Therefore a network node can and should not attempt to impact version selection and
+as such version downgrade is considered an attack.
```

Comment:
There are a number of active attacks against a QUIC handshake.  All of them result in connection failures.  This will be no different.  I don't really see the point of isolating this, except to note that the specific methods involved are not yet defined.  Dropping this last sentence would be a start, but I would reconsider the entire paragraph.

Reply:
```suggestion
Therefore a network node should not attempt to impact version selection, as version
downgrade may result in connection failure.
```

Reply:
I wanted to explicitly about version downgrades because I was thinking that the people who do version filtering might easily get the idea that version downgrade could be a solution to some of their problems and would not think of it as an attack. I agree that this is true for any handshake "attack" but I would like to be as explicit as possible about what we recommend and not recommend in this doc for cases where we can already foresee problems.